eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -38,26 +29,52 @@ jobs:
       - template: /eng/pipelines/templates/steps/vcpkg.yml
         parameters:
           DependenciesVariableName: vcpkg.deps
+          
+      - template: /eng/common/TestResources/deploy-test-resources.yml
+        parameters:
+          ServiceDirectory: '.'
+
+      - pwsh: |
+          Uninstall-AzureRm
```

Comment:
I would suggest adding these steps into your script as well.  I also suggest using your Powershell task directly instead of pwsh to better enable debugging, see https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/2/Azure-DevOps-Pipeline-Guidance?anchor=devops-tasks-vs-pwsh-script-shortcut for some guidance.